src/js/server/api.js

```diff
@@ -142,27 +116,54 @@ fluid.defaults("gpii.express.user.api", {
         verify: {
             type:     "gpii.express.user.api.verify",
             priority: "after:session"
+        },
+        docs: {
+            type:     "gpii.express.api.docs.router",
+            priority: "last"
         }
     }
 });
 
-// An instance of `gpii.express.user.api` that already has the required session middleware wired in
-//
-// Generally you will want to start with the base grade and provide your own.
-fluid.defaults("gpii.express.user.api.hasMiddleware", {
-    gradeNames: ["gpii.express.user.api"],
+// A mix-in grade to add the required session middleware.  You can mix this in with the api itself, but generally you
```

Comment:
I find this comment quite unclear. What does "the required session middleware" do? What does the "loginRequired" grade do?